tst/EnergyPlus/unit/HVACFourPipeBeam.unit.cc

```diff
@@ -1689,8 +1695,8 @@ TEST_F(EnergyPlusFixture, Beam_sizeandSimulateOneZone)
 
     EXPECT_DOUBLE_EQ(DataLoopNode::Node(15).Temp, 14.0);
     EXPECT_DOUBLE_EQ(DataLoopNode::Node(15).MassFlowRate, 0.0);
-    EXPECT_NEAR(DataLoopNode::Node(39).Temp, 35.064466069323743, 0.00001);
```

Comment:
These are definitely the types of changes I was expecting.  It's awesome to see the unit tests in action like this.